concent_api/core/models.py

```diff
@@ -566,6 +566,53 @@ def _current_state_enum(self) -> 'SubtaskState':
         return Subtask.SubtaskState[self._current_state_name]  # type: ignore
 
 
+class PaymentInfo(Model):
+    """
+    Stores the information needed to construt a PaymentCommitted message
+    """
```

Comment:
Please put changes like moving this class to a new place in a separate commit. Otherwise it's hard to see what really changed.

Reply:
I think the whole move is not necessary, the same effect can be achieved when wrapping class name in a string in relation field.

Reply:
Yes, I wrapped the name in string and everything is OK- thanks, didn't know about it. @cameel I dont't move the classes, so separate commit is not needed any more.